server.js

```diff
@@ -103,6 +104,7 @@ const connectSrc = [
   "https://code.cdn.mozilla.net/fonts/",
   "https://www.google-analytics.com",
   "https://accounts.firefox.com",
+  "https://accounts.stage.mozaws.net/metrics-flow",
```

Comment:
Hmm ... makes me a little nervous. But we can remove this when the experiment is done, yes?